src/kata6.joinNames.js

```diff
@@ -1,5 +1,11 @@
 const joinNames = (namesObj) => {
+ let names = '';
+ for (let i = 0; i < namesObj.length - 1; i++) {
+  names += namesObj[i].name + ', '
+ }
+  names += namesObj[namesObj.length - 1].name;
 
+  return names.replace(/,(?=[^,]*$)/,' &')
```

Comment:
You could just put the & in on line 4 in place of the comma, and remove the need for this regexp?